README.md

```diff
@@ -18,16 +18,16 @@ Click on the name of a plugin or module to view that content's documentation:
   - **Filter Plugins**:
   - **Inventory Source**:
   - **Callback Plugins**:
-    - [grafana_annotations](#)
+    - [grafana_annotations](https://docs.ansible.com/ansible/latest/plugins/callback/grafana_annotations.html)
```

Comment:
FYI these docs are built from whats in Ansible/Ansible:stable-2.9

Collections docs is a week or two out.

Reply:
That was my next question ! I was about to check if I could self host the pages somewhere.
I think it can wait a week or two.